addons/medical_treatment/functions/fnc_handleBandageOpening.sqf

```diff
@@ -124,7 +124,7 @@ if (random 1 <= _reopeningChance * GVAR(woundReopenChance)) then {
                 // Re-add trauma and damage visuals
                 if (GVAR(clearTrauma) == 2) then {
                     private _injuryDamage = (_selectedInjury select 4) * _impact;
-                    private _bodyPartDamage = _target getVariable [QEGVAR(medical,bodyPartDamage), [0,0,0,0,0,0]];
+                    private _bodyPartDamage = _target getVariable [QEGVAR(medical,bodyPartDamage), [0,0,0,0,0,0,0,0,0,0,0,0]];
```

Comment:
Looking at how frequently the default state array of body parts damage is being used would be nice to move it to a macro.

Reply:
Ive attempted that, hopefully its correct